### PR TITLE
[MC-159] POC - SAMPLE read an write storage

### PR DESCRIPTION
**Pull request by eliasscaceres**
## What did you implement:

Closes #XXXXX

## Todos:

_**Note: Run `npm run test-ci` to run all validation checks on proposed changes**_

- [ ] Write tests and confirm existing functionality is not broken.
       **Validate via `npm test`**
- [ ] Write documentation
- [ ] Ensure there are no lint errors.
       **Validate via `npm run lint-updated`**
       _Note: Some reported issues can be automatically fixed by running `npm run lint:fix`_
- [ ] Ensure introduced changes match Prettier formatting.
       **Validate via `npm run prettier-check-updated`**
       _Note: All reported issues can be automatically fixed by running `npm run prettify-updated`_
- [ ] Make sure code coverage hasn't dropped
- [ ] Provide verification config / commands / resources
- [ ] Enable "Allow edits from maintainers" for this PR
- [ ] Update the messages below

**_Is this ready for review?:_** NO
**_Is it a breaking change?:_** NO

### CODE REVIEW

File: samples/gcp-storage/.gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+node_modules
+
+# Serverless directories
+.serverless
+
+coverage
+
+src/multicloud-gcp
+
+.gcloud
```
Comment (FabioOrlinski) [inline samples/gcp-storage/.gitignore:11]: It seems that this exclusion is not working, could you remove it? (or maybe fix it if it´s necessary)

File: samples/gcp-storage/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "hello-world",
```
Comment (FabioOrlinski) [inline samples/gcp-storage/package.json:2]: Why the name is "hello-world"? The same happens to the handler

File: samples/gcp-storage/src/hello.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+const { App, HTTPBindingMiddleware, StorageMiddleware } = require('core');
+const { GcpModule } = require('gcp');
+
+const app = new App(new GcpModule());
+
+async function streamToString(readableStream) {
```
Comment (FabioOrlinski) [inline samples/gcp-storage/src/hello.js:6]: Perhaps you could move this function to another file? To focus on the write and read functions of the GCP Implementation

File: gcp/src/services/gcpStorage.ts [hunk 4]
```diff
@@ -21,7 +22,15 @@ export class GcpStorage implements CloudStorage {
    * Initialize new gcp storage service
    */
   public constructor() {
-    this.storage = new Storage();
+    this.storage = new Storage({
+      projectId: process.env.projectId,
+      credentials: {
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        private_key: process.env.privateKey,
+        // eslint-disable-next-line @typescript-eslint/camelcase
```
[the remark this answers is not in the document]
Comment (eliasscaceres) [reply gcp/src/services/gcpStorage.ts:30]: I think that is not a good idea to change the eslint rules for all the module.

**Pull request closed without merging**